app/controllers/nominations/nominate_induction_coordinator_controller.rb

```diff
@@ -26,12 +26,15 @@ def new
   def create
     load_nominate_induction_tutor_form
 
-    render :new and return unless @nominate_induction_tutor_form.valid?
-
-    @nominate_induction_tutor_form.save!
-    redirect_to nominate_school_lead_success_nominate_induction_coordinator_path
-  rescue UserExistsError
-    redirect_to email_used_nominate_induction_coordinator_path
+    if @nominate_induction_tutor_form.valid?
+      CreateInductionTutor.call(school: @nominate_induction_tutor_form.school,
+                                email: @nominate_induction_tutor_form.email,
+                                full_name: @nominate_induction_tutor_form.full_name)
```

Comment:
Eh, in my personal opinion passing all the data of one object to another object is a good indication that the functionality really wants to live on the first object.

And the `call` includes `save!` which could in principle throw an exception - so the form will need to be tightly logically coupled with the service - if we change what we do within the `call` so it throws with more cases, we will need to change the form validation too.

I guess all form-service separation has to deal with that problem, and maybe the cost is worth it. I'm not convinced, but I won't make you rewrite it :P

Reply:
Hmm I don't think they are tightly coupled, what we are doing is trying to separate the concerns so that the form object is concerned with validating the user input and the service object is concerned with all the all the tasks related to creating an induction tutor.  The service object can be used completely independently from the form object

Reply:
It can be used completely independently, but it will throw an error whenever it gets data that it doesn't expect. And the form has to make sure that the data is what the service expects.

It might well be that that is the reason why Rails chose to do validation and saving in the same place, on a model :P

Reply:
Where's the trollface emoji?